### ISSUE TITLE
[2021/09/23]推薦・機械学習勉強会

### ISSUE THREAD

**Issue by yu-ya4** [wantedly/machine-learning-round-table#117]
## Why

推薦・機械学習勉強会は、推薦や機械学習、その周辺技術を通じてサービスを改善することにモチベーションのある人達の集まりです。ニュースやブログから論文まで、気になったものについてお互い共有しましょう！

発信のため、ここは **public** にしてあります。外部からの参加をご希望の方は松村(https://twitter.com/yu__ya4) まで DM を送るか、Wantedly Visit の募集（https://www.wantedly.com/projects/391912） よりご連絡ください！

## What

Wantedly では隔週木曜日に

- 推薦の評価指標について議論したい
- 〇〇っていうライブラリ / フレームワークを導入してみたい
- 他社の基盤事例をみんなにシェアして自社の基盤開発に活かしたい
- もっと推薦を良くするためにどんなものが必要か議論したい

といった話をする「推薦・機械学習勉強会」を開催しています。
この ISSUE はその会で話すネタを共有するための場所です。

話したいことがある人はここにコメントしましょう！
会の間に話した内容もここにメモしましょう！

prev: #115

**Comment by hakubishin3** [comment]
## [Vision Transformer のしくみ](https://speakerdeck.com/himidev/vision-transformerfalsesikumi)
- 事前説明としてTransformerの紹介を用意しつつ、Vision Transformerの概要や応用例を丁寧に紹介している資料
- atmaCup11回ではスモールデータだったが ViT+DINO が強かったのを思い出した https://www.guruguru.science/competitions/17/discussions/d4a737a1-2f39-4c1e-9ac9-1b52c1b419e2/

## [facebookresearch/torcharrow](https://github.com/facebookresearch/torcharrow)
- PytorchのDataFrameライブラリ
- streaming-friendly API ?
- GPUバックエンドはlibcudf
- CPUでも良いパフォーマンスになるらしい
- チュートリアル: https://github.com/facebookresearch/torcharrow/blob/main/tutorial/tutorial.ipynb
- ベータ版リリースは2022年初頭の予定

## [自然言語処理の分散表現(Word2Vec,fastText)の課題](http://aiweeklynews.com/archives/55808263.html)
- 分散表現の学習時に生じる問題を体系的に整理されている

**Comment by zerebom** [comment]
## [PYTORCH PERFORMANCE TUNING GUIDE](https://tigress-web.princeton.edu/~jdh4/PyTorchPerformanceTuningGuide_GTC2021.pdf)
Pytorch高速化テクニック集。

例
- DataLoaderのnum_workersを指定して並列処理にしよう
- BatchNormに入力する時、nn.Conv(2/3/4)dのbiasはFalseにしよう
- GPU用のTensorを作る時、.cuda()で作るんじゃなくて、直接device=cudaを指定しよう

## [暗黙的フィードバックのための逐次行列分解ライブラリを公開しました](https://tech.uzabase.com/entry/2021/09/28/163623)
ユーザーベースのアルゴリズムチームの方の記事。
Numba & Pure Pythonで開発しやすく高速な、逐次パラメータ更新を行う事ができる行列分解ライブラリ"eALS"を公開。
これにより、速報ニュースのベクトルを素早く計算することが可能になった。


## [論文の書き方 - 主にメンタル面の対処を中心として](https://speakerdeck.com/kikuzo/lun-wen-falseshu-kifang-zhu-nimentarumian-falsedui-chu-wozhong-xin-tosite)
査読あり学会論文の書き方について紹介したスライド。
執筆中によくあるメンタル面の悩みについて厚く書いてあって面白かったです。技術ブログ等を書くときも参考にしたいです。

**Comment by nogawanogawa** [comment]
## [Replay推定量を勉強する](https://www.nogawanogawa.com/entry/replay_estimator)

推定量の中でベーシックなものを試してみた。

## [BigQueryで始めるt検定](https://www.nogawanogawa.com/entry/bq_ttest)

BigQueryでt検定をやってみた次第です。
